### ISSUE TITLE
.npz flow score file for combining your method with TSN.

### ISSUE THREAD

**Issue by ZealACMer** [chaoyuaw/pytorch-coviar#64]
For facilitating the reproducibility of your proposed method, could you please share the .npz optical flow score file you used to combine Coviar with TSN. It is very important for the result reproducibility. Thank you very much. Looking forward for your reply.

**Comment by ZealACMer** [comment]
I tried very hard to figure out how to generate the test file which is same as Coviar meantime the test file can be used to test the experimental results on TSN, but without success. I am so grateful if you can provide us the .npz file you used to combine your results with TSN for obtaining the final classification performance, thank you very very much.

**Comment by chaoyuaw** [comment]
Hi @ZealACMer , unfortunately I didn't keep the optical flow stream output results. 

Just to make sure I understand your questions correctly, were you able to get reasonable optical-flow-stream performance, but just not able to combine with CoViAR outputs? I think I'm a bit confused about what you mean by ""can be used to test the experimental results on TSN"".

In my humble opinions, for the sake of fair comparison, you can combine CoViAR with any other optical-flow-stream predictions, and compare with any other methods that are also combined with the same optical-flow-stream predictions. While this might not produce exactly the same accuracy as the CoViAR+Flow accuracy in paper, I think that would still be very valid and useful results.

**Comment by ZealACMer** [comment]
Thank you so much for your prompt reply. Sorry for the inappropriate expression, "can be used to test the experimental results on TSN" should be modified to "can be used to combine the experimental results of TSN". My question is have you obtained the combined results (TSN + Coviar) by modifying the format of the score files obtained in article "Temporal Segment Networks: Towards Good
Practices for Deep Action Recognition" (Are there any tips and tricks here? I still don't figure out how to combine the results of hmdb51. ) or you have retrained the aritecture in TSN by your own generated test file to obtain the classification score files, so these score files can be fused with your corresponding score files by using "combine.py" in this repository directly.
I am stucked by the problem for a long time. I deeply appreciate your kind help.
Reference:
TSN hmdb51 test split: https://github.com/yjxiong/temporal-segment-networks/tree/master/data/hmdb51_splits
Coviar hmdb51 test split: https://github.com/chaoyuaw/pytorch-coviar/tree/master/data/datalists

**Comment by chaoyuaw** [comment]
Hi @ZealACMer , I trained the optical-flow-TSN-model myself, without using the scores provided by TSN authors. 

The order of videos in my score file is based on the order defined at https://github.com/chaoyuaw/pytorch-coviar/blob/master/dataset.py#L80
namely, the order defined in "video_list". 

If you would like to use the scores from TSN repository: 
I'm not familiar with TSN repository's format, but I think the order of videos in their score file can be different from ours. Thus I guess you'll need to change the order of their score output (through matching video names) and make them into our format. 

Another thing you might want to check is to see if they define "scores" as the "logits" or the "softmax probabilities". You might want to make sure that they are consistent.  Also, different architectures might give output activations of very different scales, so you might need a different "weight" when combining the scores.

**Comment by ZealACMer** [comment]
Thank you very much for your kind help. Thanks a lot.
